addons/mail/static/src/core/common/quick_reaction_menu.js

```diff
@@ -27,13 +26,17 @@ export class QuickReactionMenu extends Component {
     setup() {
         this.toggle = useRef("toggle");
         this.store = useState(useService("mail.store"));
-        this.picker = usePopover(EmojiPicker, {
-            position: "bottom-end",
-            popoverClass: "o-mail-QuickReactionMenu-pickerPopover shadow-sm",
-            animation: false,
-            onPositioned: (el, { direction, variant }) =>
-                el.classList.add(`o-popover--${direction[0]}${variant[0]}`),
-        });
+        this.picker = useEmojiPicker(
```

Comment:
Unrelated to PR but `useEmojiPicker` is currently not typed in regards to its params.